udf/mrn_udf_highlight_html.cpp

```diff
@@ -46,13 +47,20 @@ typedef struct st_mrn_highlight_html_info
     bool used;
     grn_obj *table;
     grn_obj *default_column;
+    const char* query;
+    unsigned long query_length;
```

Comment:
It seems that we don't need to add them.

Reply:
Fixed.

I have changed the specification of `AS query` like below as I have written in the description of this PR.

> From this change, AS query also can be specified at anywhere after the second argument.

So we should use them instead of `args->args[1]` and `args->lengths[1]` when  `AS query` is specified like the following commit.

https://github.com/mroonga/mroonga/pull/539/commits/663293359d9891918aee252175a919eb0369909d